deployments/common/image/common-env/docs.pip

```diff
@@ -1,5 +1,5 @@
 matplotlib
-sphinx
+sphinx~=5.3.0
```

Comment:
Wondering if we should pin in common-env or not since it's shared by all deployments;  alternative build-hints.pip.   Guess pinning in common-env does keep base + mission kernels consistent,  I'm just wondering if it's coming from a framework deficiency or is just outright the better choice.   The property of ONLY worrying about build-hints.pip to build each kernel makes them easier to maintain individually.   Seems like the minimum requirement for the pinning is keeping base + mission kernels consistent within one deployment,  not keeping all kernels of all deployments consistent which is more likely to be impossible.   No ironclad opinion personally,  just making sure we're consciously making this choice as "easiest overall."

Reply:
Good point. This was an interesting case due to some dependency resolution pointing to alpha versions of other packages, which I assume will be resolved during the next package updates.  

After reading your comments I believe it makes more sense to only do it at the environment level, I will be updating the PR, Thanks!

Reply:
Removed the pinning from common